src/product-card.css

```diff
@@ -0,0 +1,84 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
It is a good practice to add `box-sizing: border-box;` on all elements `* {...}`